packages/vcvrack/vcvrack.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+  <mime-type type="application/x-vcvrack-patch">
+    <comment>VCV Rack Patch</comment>
```

Comment:
I'm open to suggestions here, since the type is likely to never change again

Reply:
There's also [an official](https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xml) `+zstd` suffix. Maybe the objectively best choice is `application/vnd.vcvrack.patch+zstd`.

Reply:
after finding [this](https://www.ibm.com/docs/en/sc-and-ds/8.4.0?topic=mt-adding-mime-type-mappings) I'm also adding a `x-` prefix, so now it's `application/x-vnd.vcvrack.patch+zstd` 😅